src/guild/Player.ts

```diff
@@ -576,6 +581,7 @@ export class Player extends EventEmitter {
     public onLavalinkMessage(json: any): void {
         if (json.op === OPCodes.PLAYER_UPDATE) {
             this.position = json.state.position;
+            this.lavalinkState = json.state.connected;
```

Comment:
This is player connected state, missmatch with lavalinkState. should'nt playerState?